File: letter/ru/Ь.py
from common import *

expr = Eq(
    (x + S(1) / 2)
    * ((x + S(1) / 2) ** 2 + 4 * y ** 2 + 4 * y)
    * sqrt(
        (x + S(1) / 2 + eps) /
        (1 - Abs(y))
    )
    ,
    0
)
size = 1

File: letter/ru/Ь.js
import { Letter } from '../../Letter.js';

const expr = '(x + 1/2) * ((x + 1/2)^2 + 4*y^2 + 4*y) * sqrt((x + 1/2 + 1/10) / (1 - abs(y)))';

export const letter = new Letter(expr, 1);
